README.md

```diff
@@ -448,3 +448,54 @@ JSON formatted Zone Report
     }]
 }
 ```
+
+
+### /logicalpath/repl
+Replicates a data object into some resource
+
+**Method**: POST
+
+**Parameters**:
+- all: If selected, updates all stale copies.
+- recursive:  Required to repl a collection. Replicates the whole subtree. Does nothing if used on a data object.
+- num-threads: The number of threads to use for replication.
+- repl-num: Specifies the particular replicate to copy from. Typically not needed.
+- dest-resource: Resource to which to replicate the data object or collection.
+- logical-path: The logical path of the data object or collection to replicate.t
+
+
+**Example CURL command**:
+```
+export TARGET_PATH=/tempZone/home/rods/hello.cpp
+curl -X POST -H "Authorization: ${TOKEN}" 'http://localhost/irods-rest/0.9.1/resource?logical-path=$TARGET_PATH'
+```
+
+### /logicalpath/trim
+
+**Method**: DELETE
+**Parameters**:
+- recursive: Required to trim a collection. Trims the whole subtree. Does nothing if used on a data object.
+- dryrun: If set, no copy will actually be trimmed.
+- age: The minimum age in minutes for a data object to be a candidate for trimmed. If a data object is older, nothing will happen to it.
+- num: The minimum number of copies to leave after trimming. Defaults to 2.
+- repl-num: Determines the replica to trim.
+- src-resc: If specified, only copies on this resource will be candidates for trimming.
+- admin: Required for an admin to trim replicas of other users' data objects.
+
+Note that the `/admin` endpoint already exposes the ability to make
+or remove resources, in addition to other resource-related operations.
+
+**Returns**
+Nothing on success.
+On failure, returns a JSON object describing the error. For example
+this CURL command passes an empty string as the target path:
+
+```
+curl -X DELETE -H "Authorization: ${TOKEN}" http://localhost:80/irods-rest/0.9.0/logicalpath/trim?$logical-path=
+```
+
+And produces the following output:
+
+```
+{"error_code":-808000,"error_message":"Could not find the logical path you entered. You may have passed an empty string."}
+```
```

Comment:
Given that we've defined the structure of the error JSON object, we probably don't need to demonstrate what failure looks like.

Instead, we just provide an example demonstrating how to invoke the endpoint.